src/lib/viewers/controls/media/TimeControls.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { bdlBoxBlue, bdlGray62, white } from 'box-ui-elements/es/styles/variables';
+import SliderControl from '../slider';
+import './TimeControls.scss';
+
+export type Props = {
+    bufferedRange?: TimeRanges;
+    currentTime?: number;
+    durationTime?: number;
+    onTimeChange: (volume: number) => void;
+};
+
+export const round = (value: number): number => {
+    return +value.toFixed(4);
```

Comment:
Could you explain what the `+` symbol is for here?

Reply:
It's a shortcut to convert a string to a number, as the former is what `toFixed` returns.